src/Services/ClinicalNotesService.php

```diff
@@ -344,17 +344,19 @@ public function isValidClinicalNoteCode($code)
         return !empty($options);
     }
 
-    public function getClinicalNoteTypes()
+    public function getClinicalNoteTypes($includeInactive = false)
```

Comment:
I guess this is on me however I fully expected the service to return only active items from the list. Really, why would you need inactive items ever!
Nice save robert, thanks.